tests/functional/processes/frbr/test_cluster_process.py

```diff
@@ -12,23 +13,17 @@ def test_cluster_process(db_manager, unclustered_record_uuid):
     
     db_manager.session.commit()
 
-    clustered_record = db_manager.session.query(Record).filter(Record.uuid == unclustered_record_uuid).first()
+    assert_record_clustered(record_uuid=unclustered_record_uuid, db_manager=db_manager)
 
-    assert clustered_record.cluster_status == True
-    
-    frbrized_model = (
-        db_manager.session.query(Item, Edition, Work)
-            .join(Edition, Edition.id == Item.edition_id)
-            .join(Work, Work.id == Edition.work_id)
-            .filter(Item.record_id == clustered_record.id)
-            .first()
-    )
+@pytest.mark.skip
+def test_cluster_record(db_manager, unclustered_record_uuid):
```

Comment:
we may want to add a separate record because I think it could get clustered by the other tests right?